src/Framework/TestSuite.php

```diff
@@ -718,4 +639,114 @@ private function invokeMethodsAfterLastTest(Event\Emitter $emitter): void
             );
         }
     }
+
+    /**
+     * @param ReflectionClass<TestCase> $class
+     * @param list<non-empty-string>    $groups
+     *
+     * @throws Exception
+     */
+    protected function addTestMethod(ReflectionClass $class, ReflectionMethod $method, array $groups): void
```

Comment:
why have this methods been moved in this file? this makes the actual diff hard to read

Reply:
It was done by CS-fixer. Only changed the access type.

Reply:
I see, thx. I have this CS-fixer rule :)

Reply:
@sebastianbergmann wdyt about this CS fixer rule? :)

Reply:
I agree that there is a conflict of objectives at play here: one the one hand, I want methods to be ordered in a certain way. On the other hand, the proposed changes should be readable. In this case the method ordering gets in the way of patch readability because the visitbility of a method is changed.

Reply:
Ok, how can I revert it or close this branch and add the modifications without CS fixer?